README.md

```diff
@@ -55,7 +55,7 @@ Haystack is built in a modular fashion so that you can combine the best technolo
 
 |                                                                                               |                                                                                                                                                                                                                                                   |
 | --------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| :ledger: [Docs](https://haystack.deepset.ai/overview/intro)                                   | Overview, Components, Guides, API documentation                                                                                                                                                                                                   |
+| :ledger: [Docs](https://docs.haystack.deepset.ai/docs)                                   | Components, Pipeline Nodes, Guides, API Reference                                                                                                                                                                                                   |
```

Comment:
I'm not sure what the difference is but the link to the intro topic is actually https://docs.haystack.deepset.ai/docs/intro
though the one you added seems to work OK too.

Reply:
My 2c, that should point to https://docs.haystack.deepset.ai/

Reply:
I was going to suggest let's wait to have the landing page designed to point to  https://docs.haystack.deepset.ai/ so I would say https://docs.haystack.deepset.ai/docs/intro or https://docs.haystack.deepset.ai/docs (both point to the same thing @agnieszka-m )

Reply:
let's point to https://docs.haystack.deepset.ai/